src/main/java/com/avispl/symphony/dal/communicator/nec/multisync/NECMultisyncDevice.java

```diff
@@ -171,11 +199,6 @@ private int getTemperature() throws Exception{
         //setting the sensor to retreive the temperature from
         send(NECMultisyncUtils.buildSendString((byte)monitorID,MSG_TYPE_SET,CMD_SET_SENSOR,SENSOR_1));
 
-        //wait a sec to allow the display to settle
-        synchronized(this) {//synchronized block
-            Thread.sleep(1000);
```

Comment:
There must be a reason to wait here, is it ok to remove the wait?

Reply:
Seems like this one was made specifically for the emergency delivery situation (after power on/off action) but you are right, I could've made a quick assumption with this one even though we didn't see the error related to temperature sensor on the devices available on int (the error was previously there before the local statistics was stored). 
To make sure and avoid any issues related - I have asked Jonathan about the exact issue 1s sleep was meant to solve in the past.